dagstore/dagstorewrapper.go

```diff
@@ -110,8 +112,23 @@ func (ds *dagStoreWrapper) LoadShard(ctx context.Context, pieceCid cid.Cid) (car
 	key := shard.KeyFromCID(pieceCid)
 	resch := make(chan dagstore.ShardResult, 1)
 	err := ds.dagStore.AcquireShard(ctx, key, resch, dagstore.AcquireOpts{})
+
 	if err != nil {
-		return nil, xerrors.Errorf("failed to schedule acquire shard for piece CID %s: %w", pieceCid, err)
+		if errors.Unwrap(err) != dagstore.ErrShardUnknown {
```

Comment:
nit: can we use `xerrors.Is(err, dagstore.ErrShardUnknown)?`

Reply:
@dirkmc Is there a semantic difference between the two ?

Reply:
Done.